memor/prompt.py

```diff
@@ -115,23 +107,20 @@ def __copy__(self):
         result.__dict__.update(self.__dict__)
         return result
 
-    def copy(self):
+    def copy(self) -> Any:
```

Comment:
Prompt

Reply:
Since it's in the same class it can not be used.